cmd/kubectl-testkube/commands/tests/executions.go

```diff
@@ -2,41 +2,50 @@ package tests
 
 import (
 	"os"
+	"strings"
 
 	"github.com/kubeshop/testkube/cmd/kubectl-testkube/commands/common"
+	"github.com/kubeshop/testkube/cmd/kubectl-testkube/commands/common/render"
+	"github.com/kubeshop/testkube/cmd/kubectl-testkube/commands/tests/renderer"
 	"github.com/kubeshop/testkube/pkg/ui"
 	"github.com/spf13/cobra"
```

Comment:
Do you think it will be a good idea to sort packages in the following order: builtin, external and then ours?

Reply:
I'm using goimports and it sorts them don't want to do it manually